src/androidTest/java/de/dennisguse/opentracks/LocaleRule.java

```diff
@@ -0,0 +1,57 @@
+package de.dennisguse.opentracks;
+
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.util.DisplayMetrics;
+
+import androidx.test.core.app.ApplicationProvider;
+
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+import java.util.Locale;
+
+public class LocaleRule implements TestRule {
+
+    private final Locale[] mLocales;
+    private Locale mDeviceLocale;
+
+
+    public LocaleRule(Locale... locales) {
+        mLocales = locales;
```

Comment:
Assert that the array is not null and length > 0?

Reply:
Ok, then the check further down can be removed